CHANGELOG.md

```diff
@@ -29,12 +29,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
   * `MultiQC v1.7`
   * `STAR aligner v2.7.0f`
 
-### Added
+### Added - [1.1.0]
```

Comment:
Why adding version? This is not following the guidelines.

Reply:
I had mardownlint issues, but now I know that it was coming from another place.
removing all that.